src/sparse_approximations.jl

```diff
@@ -288,3 +288,7 @@ end
 function tr_Cf_invΣy(f::FiniteGP, Σy::Diagonal)
     return sum(var(f.f, f.x) ./ diag(Σy))
 end
+
+function tr_Cf_invΣy(f::FiniteGP, Σy::ScalMat)
+    return sum(var(f.f, f.x) / Σy)
```

Comment:
This is just

```suggestion
    return sum(var(f.f, f.x)) / Σy.value
```
or, without accessing internal fields that could possibly be changed and then break AbstractGPs,

```suggestion
    return sum(var(f.f, f.x)) / first(Σy)
```